install_config/configuring_local.adoc

```diff
@@ -132,35 +139,47 @@ Before starting the provisioner, mount all local devices and create a ConfigMap
 with storage classes and their directories.
 ====
 
+
 Install the local provisioner from the link:https://raw.githubusercontent.com/jsafrane/origin/local-storage/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml[*_local-storage-provisioner-template.yaml_*] file.
 
-. Create a service account that allows running pods as a root user and use HostPath volumes:
+. Create a service account that allows running pods as a root user, use HostPath volumes and use any SELinux context to be able to monitor, manage and clean local volumes:
 +
 [source, bash]
 ----
 $ oc create serviceaccount local-storage-admin
-$ oc adm policy add-scc-to-user hostmount-anyuid -z local-storage-admin
+$ oc adm policy add-scc-to-user privileged -z local-storage-admin
 ----
-Root privileges are required for the provisioner pod for allowing it to delete
-content on local volumes. HostPath is required to access the
+Root privileges and any SELinux context are required for the provisioner pod for allowing it to delete
```

Comment:
To allow the provisioner pod to delete content on local volumes created by any pod, root privileges and any SELinux context are required.